model/account.go

```diff
@@ -0,0 +1,10 @@
+package model
+
+type Account struct {
+	ID       uint   `gorm:"primary_key"`
+	User     string `gorm:"not null;unique"`
+	Password string `gorm:"not null"`
```

Comment:
I agree we shouldn't store plain password. So, I think we should rename this field to something like `PasswordEncrypted` + `Salt` and have a "setter" to it, which would receive it plaintext and store it encrypted.